qemu/tests/uefi_check_secure_MOR.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+
+from avocado.utils import process
+
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Please make sure the guest installed with signed driver
+    Verify Secure MOR control feature using Device Guard tool in Windows guest:
+
+    1) Boot up a guest.
+    2) Check if Secure Boot is enable.
+    3) Download Device Guard and copy to guest.
+    4) Enable Device Guard and check the output.
+    5) Reboot guest.
+    5) Run Device Guard and check the output.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def execute_powershell_command(command):
+        status, output = session.cmd_status_output(command)
+        logging.info("command:%s" % command)
+        logging.info("command output:%s" % output)
+        if status != 0:
+            test.fail("execute command fail: %s" % output)
+        return output
+    timeout = int(params.get("timeout", 360))
+    params['ovmf_vars_filename'] = 'OVMF_VARS.secboot.fd'
+    params['start_vm'] = 'yes'
+    params['cpu_model_flags'] = ',hv-passthrough'
+    env_process.preprocess_vm(test, params, env, params['main_vm'])
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_serial_login(timeout=timeout)
+
+    check_cmd = params['check_secure_boot_enabled_cmd']
+    dgreadiness_tool_path = params['dgreadiness_tool_path']
+    executionPolicy_command = params['executionPolicy_command']
+    enable_command = params['enable_command']
+    ready_command = params['ready_command']
+
+    try:
+        status, output = session.cmd_status_output(check_cmd)
```

Comment:
dup code with execute_powershell_command? You might wanna highlight an different failure msg here, but we can still call execute_powershell_command if we can read 'secure boot is not enabled' info from the output, what's your opition?